docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -80,17 +82,19 @@ public class AdminResource {
 }
 ----
 
-Injection of `JsonWebToken` is supported in `@ApplicationScoped`, `@Singleton` and `@RequestScoped` scopes however the use of `@RequestScoped` is required if the individual claims are injected as simple types, please see xref:security-jwt.adoc#supported-injection-scopes[Support Injection Scopes for JsonWebToken and Claims] for more details.
+Injection of `JsonWebToken` is supported in `@ApplicationScoped`, `@Singleton`, and `@RequestScoped` scopes.
+However, the use of `@RequestScoped` is required if the individual claims are injected as simple types.
+For more information, see the xref:security-jwt.adoc#supported-injection-scopes[Support Injection Scopes for JsonWebToken and Claims]  section of the Quarkus "Using JWT RBAC" guide.
```

Comment:
Minor/non-urgent suggestion: Should the capitalization of the URL text be in sentence case.

Reply:
@sheilamjones  Fixed. Good catch. Thanks :100: 
_Your comment also triggered a review of the capitalization in some level 3 section headings. Fixed a few of those too._